call.go

```diff
@@ -138,6 +138,8 @@ func sendRequest(ctx context.Context, dopts dialOptions, compressor Compressor,
 
 // Invoke sends the RPC request on the wire and returns after response is
 // received.  This is typically called by generated code.
+//
+// All errors returned by Invoke are status errors.
```

Comment:
How about "are errors compatible with the status package"?

Reply:
Done